tests/test_core.py

```diff
@@ -4564,6 +4564,7 @@ def test_fs_trackingdelegate(self):
     out = path_from_root('tests', 'fs', 'test_trackingdelegate.out')
     self.do_run_from_file(src, out)
 
+  @no_mac('NODERAWFS support on mac is flaky')
```

Comment:
this also disables the non-NODERAWFS part on mac. i guess there isn't a simple way to disable just that part, though, unless we split up this test. not sure it's important enough to justify that.

Reply:
I split the test into two.